lib/Doctrine/ORM/Configuration.php

```diff
@@ -126,6 +126,33 @@ public function setProxyNamespace($ns)
     }
 
     /**
+     * Returns the umask that controls file permissions for generated proxies.
+     *
+     * @return int
+     */
+    public function getProxyUmask()
+    {
+        return isset($this->_attributes['proxyUmask']) ?
+            $this->_attributes['proxyUmask'] : 0002;
```

Comment:
Do we want to maintain the default umask here? Or is it better to rely on the default value of the `$umask` parameter in the `ProxyGenerator`?